otelcollector/configmapparser/tomlparser-scrape-interval.rb

```diff
@@ -0,0 +1,166 @@
+#!/usr/local/bin/ruby
+# frozen_string_literal: true
+
+require "tomlrb"
+if (!ENV["OS_TYPE"].nil? && ENV["OS_TYPE"].downcase == "linux")
+  require "re2"
+end
+require "yaml"
+require_relative "ConfigParseErrorLogger"
+
+LOGGING_PREFIX = "default-scrape-interval-list"
+
+@configMapMountPath = "/etc/config/settings/default-targets-scrape-interval-settings"
+@configVersion = ""
+@configSchemaVersion = ""
+
+@kubeletScrapeInterval = "30s"
```

Comment:
I think we need to add some validation to make sure the interval that they provide is a valid interval string. Ex - If they provide something like "myinterval", we should fail validation and not take those values for that job.